File: benches/mutations/variants/python/seed4.t3-unroll.py
def mix_rounds(state, seed):
    acc = seed
    acc = rotate(acc, 7)
    acc = acc ^ mash(state[0], 31)
    acc = acc + 1442695
    acc = acc ^ (acc >> 13)
    acc = acc * 636413
    acc = rotate(acc, 7)
    acc = acc ^ mash(state[1], 31)
    acc = acc + 1442695
    acc = acc ^ (acc >> 13)
    acc = acc * 636413
    acc = rotate(acc, 7)
    acc = acc ^ mash(state[2], 31)
    acc = acc + 1442695
    acc = acc ^ (acc >> 13)
    acc = acc * 636413
    return acc

File: benches/mutations/variants/rust/seed4.t3-unroll.rs
fn mix_rounds(state: &[u64], seed: u64) -> u64 {
    let mut acc = seed;
    acc = acc.rotate_left(7);
    acc ^= state[0].wrapping_mul(31);
    acc = acc.wrapping_add(1442695);
    acc ^= acc >> 13;
    acc = acc.wrapping_mul(636413);
    acc = acc.rotate_left(7);
    acc ^= state[1].wrapping_mul(31);
    acc = acc.wrapping_add(1442695);
    acc ^= acc >> 13;
    acc = acc.wrapping_mul(636413);
    acc = acc.rotate_left(7);
    acc ^= state[2].wrapping_mul(31);
    acc = acc.wrapping_add(1442695);
    acc ^= acc >> 13;
    acc = acc.wrapping_mul(636413);
    acc
}
